mo_gymnasium/envs/fruit_tree/fruit_tree.py

```diff
@@ -245,17 +247,21 @@ class FruitTreeEnv(gym.Env, EzPickle):
     """
     ## Description
 
-    Full binary tree of depth d=5,6 or 7. Every leaf contains a fruit with a value for the nutrients Protein, Carbs, Fats, Vitamins, Minerals and Water.
+    Full binary tree of depth d=5,6 or 7. Every leaf contains a fruit with
```

Comment:
I guess you added breakline for code editing? I think this might look weird on the website though (we pull the docstring directly to generate env descriptions on the website).

Reply:
Reverted 👍